tests/unit/plugins/modules/test_jenkins_build.py

```diff
@@ -12,6 +12,9 @@
 from ansible_collections.community.general.plugins.modules import jenkins_build
 
 import json
+import pytest
+
+jenkins = pytest.importorskip('jenkins')
```

Comment:
How about my suggestion of not importing this, but instead also mocking the exception class? That should be very little effort for not having another unit test dependency.

Reply:
In fact I have few knowledge on mocking so now during the unit testing without installing python-jenkins, the original jenkins_build.py file cannot recognize the jenkins.JenkinsException when it catch this exception.

Reply:
Finally managed to do so. Gained more knowledge of mocking during this period.

Reply:
Cool, thanks a lot!